File: 1/A_7_181.c
// Болібрук Влад
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

int main()
{
    int arrL = 11, j;
    int arrOfa[11] = {1, 2, 3, 4, 5, 6, 7, 8, 9, -5, -15};
    int sum = 0;

    for (int j = 0; j < arrL; j++)
    {
        int i = j * j;
        if ((arrOfa[j] % 5 == 0) && (arrOfa[j] % 2 != 0) && (arrOfa[j] < 0) && (abs(arrOfa[j]) < i))
        {
            sum += arrOfa[j];
        }
    }

    printf("task is A_7_181\n");
    printf("result  is %d\n", sum);

    return 0;
}

File: S1/3/A_7_181.java
// Болібрук Влад
public class A_7_181 {
	public static void main(String[] args) {
		int arrL = 11, j;
		int[] arrOfa = {1, 2, 3, 4, 5, 6, 7, 8, 9, -5, -15};
		int sum = 0;

		for(int j2 = 0; j2 < arrL; j2++) {
			int i = j2 * j2;
			if(arrOfa[j2] % 5 == 0 && arrOfa[j2] % 2 != 0 && arrOfa[j2] < 0 && Math.abs(arrOfa[j2]) < i) {
				sum += arrOfa[j2];
			}
		}

		System.out.println("task is A_7_181");
		System.out.println("result  is " + sum);
	}
}